tests/unit/environ_spec.lua

```diff
@@ -180,4 +187,26 @@ describe("luasnip.util.environ", function()
 		"Environ with multiline_vars incorrect type",
 		[[Environ.env_namespace("TES_T", {var={A='s', multiline_vars = 9 }})]]
 	)
+
+	local function check_builtin(var_name, test)
+		it("Test builtin " .. var_name, function()
+			assert.is_true(
+				exec_lua(
+					([=[
+					local Environ = require("luasnip.util.environ")
+                                        local env = Environ:new({pos={0, 0},  captures={"one"}, trigger="trigg"})
+                                        local result = env["%s"]
+                                        local test = %s
+                                        return test(result)
+                                        ]=]):format(
+						var_name,
+						test
+					)
+				)
+			)
+		end)
+	end
+
+	check_builtin("LS_TRIGGER", [[function(r) return r == "trigg" end]])
+	check_builtin("LS_CAPTURE1", [[function(r) return r == "one" end]])
```

Comment:
if we're going for long name, for capture group, it could be named `LS_CAPTURE_1` ?

Reply:
[«Is mir egal»](https://www.youtube.com/watch?v=YEYim54pJ00) @L3MON4D3  should decide.

Reply:
:D
I agree, it's easier to read :+1: